File: 1º Semestre/Programação de Computadores/Dev C++ exercícios/Lista de exercício 03/9 - for.cpp
/* Elabore um algoritmo que solicite ao usuário um número inteiro que indicará a quantidade
de vezes que o texto "Hello World!" será impresso na tela, um em cada linha. (for)  */

#include <stdio.h>
	
	int main(){
		int numero;
		
		printf("Informe a quandidade de vezes para imprimir o texto 'Hello World!': ");
		scanf("%d", &numero);
		
		for (int contador = 1; contador <= numero; contador++){
			printf("%3d - Hellow World!\n", contador);
		}
		
		return 0;
	}

File: 1º Semestre/Programação Python/Listas de Exercícios/Exercício 3/9 - for.py
"""9. Elabore um algoritmo que solicite ao usuário um número inteiro que indicará a quantidade
de vezes que o texto "Hello World!" será impresso na tela, um em cada linha."""

contador = 1
numero = int(input("Informe a quantidade de vezes para imprimir o texto 'Hello World': "))

for numero in range(1, numero+1):
    print("Hello World!")
